tests/sentry/api/endpoints/test_organization_metrics.py

```diff
@@ -1428,14 +1293,15 @@ def test_crash_free_percentage(self):
         assert group["totals"]["session.crashed"] == 4
         assert group["series"]["session.crash_free_rate"] == [None, None, 50, 50, 50, 50]
 
-    @with_feature(FEATURE_FLAG)
+    @freeze_time("2021-12-11 03:26:31")
```

Comment:
Careful about hardcoding a timestamp, I think this might fail if we create things outside of the retention window. At least I've seen that happen for events.

I've had success doing something like `@freeze_time((now() - timedelta(days=2)).replace(hour=3, minute=26, second=31, microsecond=0))`

Reply:
Ah yeah .. that makes sense.. had no idea .. thanks!